Added new Local Outreach and Artist Volunteer Descriptions

dchen97 opened the pull request:
Based on adjusted demand for particular roles, we decided to replace the "Design" volunteer description with an "Artist" description and the "Research" volunteer description with a "Local Outreach" member description.

hoangviet1993 commented on app/static/pages/volunteer/VolunteerProfilesService.js:
```diff
@@ -41,23 +41,26 @@ oppiaFoundationWebsite.factory('VolunteerProfilesService', [
     var volunteersByCategory = {};
     Object.keys(VOLUNTEER_INFO).forEach(function(volunteerCategory) {
+
+      console.log(volunteerCategory);
```
I don't think you need this logging statement.

dchen97 replied:
Deleted!

hoangviet1993 commented on app/static/pages/volunteer/VolunteerProfilesService.js:
```diff
@@ -41,23 +41,26 @@ oppiaFoundationWebsite.factory('VolunteerProfilesService', [
-      getDesignProfiles: function() {
-        return volunteersByCategory.design;
+      getArtistProfiles: function() {
+        return volunteersByCategory.art;
```
It's `volunteersByCategory.artist` like you defined in `VolunteerInfoDefinition.js`

dchen97 replied:
(adjusted to be art instead of artist in all references)

hoangviet1993 commented on app/static/pages/volunteer/VolunteerInfoDefinitions.js:
```diff
@@ -14,28 +14,13 @@
-      'to contribute to this meaningful cause.',
-    position: 'UX Designer'
-  }],
+  artist: [{
```
Should we change this to "Art" instead of "Artist? The other category indicates a certain field of expertise (development, local outreach, etc...)
The `position` property below should indicate clearly what the volunteer does within each specific category.

dchen97 replied:
done!

dchen97 commented:
Thanks for the review @hoangviet1993! Should be ready for another pass whenever you're ready

seanlip commented on app/static/pages/volunteer/VolunteerInfoDefinitions.js:
```diff
@@ -14,28 +14,13 @@
+  art: [{
+    fullName: 'Tuna R. Terzi',
+    profilePictureImageFilename: 'tuna.jpg',
+    professionalQuote: 'Oppia turns to my drawings into little seeds for the ' + 
```
Should "to" be struck?

dchen97 replied:
Done!

seanlip commented on app/static/pages/volunteer/VolunteerInfoDefinitions.js:
```diff
@@ -14,28 +14,13 @@
 oppiaFoundationWebsite.constant('VOLUNTEER_INFO', {
 
-  design: [{
-    fullName: 'Mark Halpin',
```
I think Mark can be kept as an artist.

dchen97 replied:
Done!

seanlip commented on app/static/pages/volunteer/VolunteerInfoDefinitions.js:
```diff
@@ -69,29 +54,6 @@ oppiaFoundationWebsite.constant('VOLUNTEER_INFO', {
-      'carefully to what they have to say!',
-    position: 'User Researcher (Boston)'
-  }, {
-    fullName: 'Kevin Lee',
```
Kevin can be kept as a developer.

dchen97 replied:
Done!

seanlip commented on app/static/pages/volunteer/VolunteerProfilesService.js:
```diff
@@ -40,24 +40,24 @@ oppiaFoundationWebsite.factory('VolunteerProfilesService', [
     };
     var volunteersByCategory = {};
-    Object.keys(VOLUNTEER_INFO).forEach(function(volunteerCategory) {
+    Object.keys(VOLUNTEER_INFO).forEach(function(volunteerCategory) {      
```
Drop trailing spaces.

dchen97 replied:
Done!

seanlip commented on app/static/pages/volunteer/work_description_templates/partnership_description.html:
```diff
@@ -0,0 +1,8 @@
+<p class="oppia-foundation-bold-text">
```
Where is this file used?

seanlip commented on app/static/pages/volunteer/VolunteerInfoDefinitions.js:
```diff
@@ -111,5 +73,13 @@ oppiaFoundationWebsite.constant('VOLUNTEER_INFO', {
     position: 'Coordinator, Lesson Creation'
+  }],
+
+  local_outreach: [{
```
Do you think it is worth adding blurbs for Yamama and Sam / Rev Bankas? This PR doesn't need to block on that but it might be nice to solicit those and add them.

dchen97 replied:
Yes, I'd love to include them once we get quotes from them!

dchen97 commented:
@seanlip - I believe I've addressed all of your comments (at least the ones I can address for now - I'll be sure to follow up on getting quotes from Yamama, Sam, and Rev. Bankas!). Ready for another pass whenever you're free!

seanlip commented:
Hi @dchen97, make sure to reply to all comments please. I think you missed this one: https://github.com/oppia/foundation-website/pull/128#discussion_r364533396 

But otherwise, LGTM! Thanks, and feel free to merge this once the Travis checks pass.
